index.js

```diff
@@ -2,14 +2,15 @@ const http = require('http')
 const rangeParser = require('range-parser')
 const mime = require('mime-types')
 const ReadyResource = require('ready-resource')
-const z32 = require('z32')
 const safetyCatch = require('safety-catch')
+const { pipelinePromise } = require('streamx')
 
 module.exports = class ServeDrive extends ReadyResource {
-  constructor (opts = {}) {
+  constructor (getDrive, releaseDrive, opts = {}) {
```

Comment:
would make these options, ie

```
new ServeDrive({
  async getDrive () {
    return drive
  }
  async releaseDrive (drive) {
  },
... opts
})
```

Reply:
`releaseDrive` can easily be an option, but I don't see how `getDrive` can be an option  (what drive would the default return?)

Reply:
Both are options now, with `getDrive` being required